backend/server/migrations/20191202140011-final-review-grade-minimum-requirement.js

```diff
@@ -1,22 +1,25 @@
-'use strict';
+
 
 module.exports = {
   up: async (queryInterface, Sequelize) => {
     await queryInterface.addColumn('Weeks', 'grade', {
-      type: Sequelize.INTEGER
+      type: Sequelize.INTEGER,
+      allowNull: true
     })
     await queryInterface.addColumn('ChecklistItems', 'minimumRequirementMetIf', {
       type: Sequelize.BOOLEAN,
-      defaultValue: false
+      defaultValue: false,
+      allowNull: false
     })
     await queryInterface.addColumn('ChecklistItems', 'minimumRequirementGradePenalty', {
       type: Sequelize.INTEGER,
-      defaultValue: 0
+      defaultValue: 1,
+      allowNull: false
```

Comment:
Do we want this column to be non-null as not all checklist items are minimum requirements?

Reply:
It should still have a number in it regardless in case it is set as a minimum requirement